Fix ingress documentation (#7424)
traefik/traefik: docs/content/routing/providers/kubernetes-ingress.md

## TLS

### Enabling TLS via HTTP Options on Entrypoint

TLS can be enabled through the [HTTP options](../entrypoints.md#tls) of an Entrypoint:

```bash tab="CLI"
# Static configuration

> -- entrypoints.websecure.address=:443
> -- entrypoints.websecure.http.tls

Reviewer: ```suggestion
--entrypoints.websecure.address=:443
--entrypoints.websecure.http.tls
```

            backend:
              serviceName: service1
              servicePort: 80

>       # Only loading certificates from Secrets, that will be used to terminate TLS traffic.

Reviewer 2: ```suggestion
Only selects which certificate(s) should be loaded from the secret, in order to terminate TLS
```

Author: @SantoDE I made the changes :-)
